src/main/resources/application.properties

```diff
@@ -1,3 +1,3 @@
 camel.component.jira.jira-url=https://issues.redhat.com/
 %test.jira-issue-lottery.access-token=ignored
-jira-issue-lottery.config-file-repo=https://github.com/The-Huginn/jira-issue-lottery
```

Comment:
I am not sure about this, perhaps each of us should keep this in `.env` file and mention this in readme

Reply:
Should I create a new issue for that? Then I can add a new commit to this PR and mention both issues.